src/main/scala/cromwell/binding/types/WdlFileType.scala

```diff
@@ -1,9 +1,5 @@
 package cromwell.binding.types
 
-import java.io.File
-
 case object WdlFileType extends WdlType {
-  def isCompatible(value: Any) = value.isInstanceOf[File]
-
   override def toString: String = "File"
```

Comment:
We could do that though there's code in WdlBinding that checks if a string from WDL source code (e.g. "Int", "String") is a WdlType then returns the corresponding WdlType object, we'd have to move that elsewhere, though maybe it's better to do an implicit class?

```scala
"Int".toWdlType
"String".toWdlType
"Array[Int]".toWdlType
```

???